index.js

```diff
@@ -4,13 +4,61 @@ var pluralize = require("pluralize");
 module.exports = function(app, handler){
 	return {
 		addResource : function(name){
-			app.use("/" + pluralize(name), function(req, res, next){
-				handler.findAll(S("_" + name).camelize().s)
-				.then(function(records){
-					res.json(records);
-				})
-				.catch(next);
-			});
+			var resourceName = S("_" + name).camelize().s;
+
+			var buildRoutes = function(name){
+				var router = require("express").Router();
+
+				router.param("id", function(req, res, next, id){
```

Comment:
Awesome, didn't think they were ever going to make that change after rejecting it earlier.

Make sure the `package.json` requires a high enough express version for that to work. It should be a peerDependency >= whatever version.